controller/adoptopenebs/common.go

```diff
@@ -47,8 +55,18 @@ func (p *Planner) formComponentOpenEBSConfig(component *unstructured.Unstructure
 	return nil
 }
 
+// formDefaultStoragePath forms the default OpenEBS directory.
+func (p *Planner) formDefaultStoragePath() error {
+	// If default storage path is not set using the environment variable OPENEBS_IO_BASE_DIR, etc.
+	if len(p.DefaultStoragePath) == 0 {
+		p.DefaultStoragePath = "/var/openebs"
```

Comment:
Are we sure env variable was the only way of overriding the default storage path?

Reply:
This is one of the ways that we have figured out we can use and we will be continuously adding different ways as fallback as we are aware or confident of them.